src/Nethermind/Nethermind.Blockchain.Test/Rewards/RewardCalculatorTests.cs

```diff
@@ -56,12 +59,14 @@ public void No_uncles()
 
         [Test, Timeout(Timeout.MaxTestTime)]
         public void Byzantium_reward_two_uncles()
-        {
+        {   
+            // Warning Ropsten Network is Deprecated
```

Comment:
for tests, we can use MainnetSpecProvider instead of Ropsten and remove this warning: "Warning Ropsten Network is Deprecated" and we can remove RopstenSpecProvider